package.json

```diff
@@ -5,10 +5,13 @@
   "scripts": {
     "lint": "tslint --project tsconfig.json",
```

Comment:
I remember that the tslint is not alive anymore. Maybe we should use eslint instead?

Reply:
Yes it is a good idea. I'll try to migrate tslint to eslint